flink-libraries/flink-table/pom.xml

```diff
@@ -92,7 +92,12 @@ under the License.
 			</exclusions>
 		</dependency>
 
-
+		<dependency>
```

Comment:
Here is what I found out. WTFPL is compatible with AL2 (https://www.apache.org/legal/resolved.html).

Maven Central lists the dependency as BSD licensed (https://search.maven.org/#artifactdetails%7Corg.reflections%7Creflections%7C0.9.10%7Cjar) which is also OK.
However, it seems that some of its transitive dependencies are LGPL licensed which would not work. So maybe we have to exclude those.

Copying a file and relicensing should be OK under WTFPL. However, we should clearly indicate the origin of the code.